src/ast.rs

```diff
@@ -304,6 +304,13 @@ pub enum ClauseGuard<Type> {
         right: Box<Self>,
     },
 
+    GreaterThan {
```

Comment:
Can you name this `GtInt`, to match the `GtInt` node elsewhere in the AST?

Reply:
Modified symbol!